sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,81 @@
+-- NAXXANAR
+-- AREATRIGGER-ID
+-- SKY
+-- 5338 roof
+-- 5339 upper chamber
+-- 5340 upper platform
+-- 5334 lower platform
+-- 5332 ground
+-- GROUND
+DELETE FROM `areatrigger_scripts` WHERE `entry` IN (5332,5334,5340,5339,5338);
+INSERT INTO `areatrigger_scripts` (`entry`, `ScriptName`) VALUES
+(5332, "SmartTrigger"),
+(5334, "SmartTrigger"),
+(5340, "SmartTrigger"),
+(5339, "SmartTrigger"),
+(5338, "SmartTrigger");
+
+DELETE FROM `smart_scripts` WHERE `source_type` = 2 AND `entryorguid` IN (5332,5334,5340,5339,5338);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(5332, 2, 0, 0, 46, 0, 100, 0, 5332, 0, 0, 0, 0, 62, 571, 0, 0, 0, 0, 0, 7, 0, 0, 0, 3733.68, 3563.25, 290.812, 3.6651, "Areatrigger - On Trigger - Teleport player"),
+(5334, 2, 0, 0, 46, 0, 100, 0, 5334, 0, 0, 0, 0, 62, 571, 0, 0, 0, 0, 0, 7, 0, 0, 0, 3802.38, 3585.95,  49.576,    0.0, "Areatrigger - On Trigger - Teleport player"),
+(5340, 2, 0, 0, 46, 0, 100, 0, 5340, 0, 0, 0, 0, 62, 571, 0, 0, 0, 0, 0, 7, 0, 0, 0, 3687.91, 3577.28, 473.342,    0.0, "Areatrigger - On Trigger - Teleport player"),
+(5340, 2, 1, 0, 46, 0, 100, 0, 5340, 0, 0, 0, 0, 62, 571, 0, 0, 0, 0, 0, 7, 0, 0, 0, 3744.80, 3570.65, 341.66,     0.6, "Areatrigger - On Trigger - Teleport player"),
+(5339, 2, 0, 0, 46, 0, 100, 0, 5339, 0, 0, 0, 0, 62, 571, 0, 0, 0, 0, 0, 7, 0, 0, 0, 3733.68, 3563.25, 290.812, 3.6651, "Areatrigger - On Trigger - Teleport player"),
+(5338, 2, 0, 0, 46, 0, 100, 0, 5338, 0, 0, 0, 0, 62, 571, 0, 0, 0, 0, 0, 7, 0, 0, 0, 3733.68, 3563.25, 290.812, 3.6651, "Areatrigger - On Trigger - Teleport player");
```

Comment:
Target type must be 8 if yo use coords.

Reply:
I based it off 
```sql
SELECT * FROM smart_scripts WHERE entryorguid IN (5187, 5190) and source_type=2;
```
(5187, 2, 0, 0, 46, 0, 100, 0, 0, 0, 0, 0, 0, 62, 571, 0, 0, 0, 0, 0, 7, 0, 0, 0, 6150.76, -1071.38, 402.715, 2.1916, Areatrigger - On Trigger - Teleport)
(5190, 2, 0, 0, 46, 0, 100, 0, 0, 0, 0, 0, 0, 62, 571, 0, 0, 0, 0, 0, 7, 0, 0, 0, 6314.19, -1758.29, 457.071, 1.6787, Areatrigger - On Trigger - Teleport)
which is the K3- Garm's Rise teleporter pair.
target_type 7 works since I want to teleport the player but is this unintended mechanics?

Reply:
Yes Keep the target 7, target 8 doesn't work in this case.